client/src/elm/Pages/Prenatal/Activity/View.elm

```diff
@@ -2101,9 +2101,7 @@ viewSymptomReviewContent language currentDate assembled data =
     , div [ class "ui full segment" ]
         [ div [ class "full content" ]
             [ div [ class "ui form symptom-review" ] <|
-                div [ class "instructions" ]
```

Comment:
@Diama1 This removes the label from both screens of activty.
It should be removed only from first screen.

![Peek 2022-08-07 12-18](https://user-images.githubusercontent.com/17882245/183284048-36a05547-a7e8-42c2-9a16-131f8a584455.gif)